rm/rm-node/src/main/resources/NodeCommandLine.properties

```diff
@@ -0,0 +1,7 @@
+#***********************************************************************************
+# *************************Node startup script Properties***************************
+#***********************************************************************************
+ns.node.jar.command.line.linux = mkdir -p /tmp/node && cd /tmp/node\n if ! type -p jre/bin/java; then wget -nv -N %javaUrl%; tar -xf jre-8u281-linux-x64.tar.gz; mv jre1.8.0_281/ jre; fi\nwget -nv -O node.jar %nodeJarUrl%\n%detachedModePrefix% jre/bin/java -jar node.jar %javaOptions% -Dpython.path=%jythonPath% -r %rmUrl% -n %nodeName% -s %nodeSourceName% -w %numberOfNodesPerInstance% -v %credentials% &
```

Comment:
I don't think it's relevant to have a javaUrl pattern while subsequent commands depend on the java package name and folder. You should remove the pattern and put the url explicitely in the script

Reply:
additionnally why is not 8u312b07 used, like in https://github.com/ow2-proactive/nodesource-addons/blob/master/infrastructures/infrastructure-common/src/main/resources/NodeSource.properties ?